# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/shadow/ObjectGroup.h
# Commit: 22854b710fbf
# Full Hash: 22854b710fbfb779ac6b19fe4c9ac2570189cec0
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 4531d5c25694 -r 22854b710fbf js/public/shadow/ObjectGroup.h
--- a/js/public/shadow/ObjectGroup.h	Fri Mar 05 19:10:06 2021 +0000
+++ b/js/public/shadow/ObjectGroup.h	Fri Mar 05 19:10:07 2021 +0000
@@ -23,7 +23,6 @@
 struct ObjectGroup {
   int* unused;
   JSObject* proto;
-  JS::Realm* realm;
 };
 
 }  // namespace shadow